deploy/kubernetes/base/node.yaml

```diff
@@ -15,21 +15,14 @@ spec:
       # Host network must be used for interaction with Workload Identity in GKE
       # since it replaces GCE Metadata Server with GKE Metadata Server. Remove
       # this requirement when issue is resolved and before any exposure of
-      # metrics ports.
-      hostNetwork: true
+      # metrics ports. But hostNetwork is not working for Windodws, might be an issue
+      # when deploying on GKE windows node. See related issue https://buganizer.corp.google.com/issues/156382455
```

Comment:
let's not reference internal issues here

Reply:
fixed